cloudformation/application-setup-basic/stacks/registry/registry.yml

```diff
@@ -1,14 +1,38 @@
 AWSTemplateFormatVersion: '2010-09-09'
 Description: Deploys a Docker container registry.
 Parameters:
-  StackName:
-    Type: String
-    Description: The name of the networking stack that
-      these resources are put into.
   RegistryName:
     Type: String
     Description: The name of the container registry
 Resources:
+
+  ECSTaskExecutionRole:
```

Comment:
New home for the role in the registry stack so that we can import the role wherever we need it.